app/validators/notices/setup/abstraction-alerts/alert-thresholds.validator.js

```diff
@@ -8,6 +8,8 @@
 
 const Joi = require('joi')
 
+const errorMessage = 'Select applicable threshold(s)'
```

Comment:
If this is a top level `const` then the variable name [should be declared in uppercase](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage)

```suggestion
const ERROR_MESSAGE = 'Select applicable threshold(s)'
```